### PR TITLE
zh: add `clean-dump-file` configuration (release-1.0)

### PR DESCRIPTION
**Pull request by lance6716**
### What is changed, added or deleted? (Required)

add configuration introduced by https://github.com/pingcap/dm/pull/804

### Which DM version(s) do your changes apply to? (Required)

- [ ] master (the latest development version, including v2.0 changes for now)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):
  https://github.com/pingcap/dm/pull/804

### CODE REVIEW

File: zh/task-configuration-file-full.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ remove-meta: false              # 是否在任务同步开始前移除该任务
 enable-heartbeat: false         # 是否开启 `heartbeat` 功能
 online-ddl-scheme: "gh-ost"     # 目前仅支持 "gh-ost" 、"pt"
 case-sensitive: false           # schema/table 是否大小写敏感
+clean-dump-file: true           # 是否清理 dump 阶段产生的文件，包括 metadata 文件、建库建表 SQL 文件以及数据导入 SQL 文件
```
Comment (WangXiangUSTC) [inline zh/task-configuration-file-full.md:33]: LGTM, 之后发版需要说明下该配置在哪个版本引入的

**Comment by WangXiangUSTC** [comment]
LGTM, 之后发版需要说明下该配置在哪个版本引入的

**Comment by lance6716** [comment]
> LGTM, 之后发版需要说明下该配置在哪个版本引入的

嗯，PR标记了tag，发版应该会检查的

File: zh/task-configuration-file-full.md (hunk 1 above)
Comment (csuzhangxc) [inline zh/task-configuration-file-full.md:33]: 这个注释里要说明一下是在 `v1.0.7` （下一个版本）引入的吗

**Pull request merged**